test/elixir_bench_web/schema/schema_test.exs

```diff
@@ -79,30 +81,403 @@ defmodule ElixirBenchWeb.SchemaTest do
        }
 
       {:ok, resp} = context.conn
-        |> post("/api", AbsintheHelpers.query_skeleton(query))
-        |> Map.get(:resp_body)
-        |> Antidote.decode
+        |> execute_query(query)
+        |> decode_response_body
 
       assert ^json_data = resp
     end
 
-    test "not break when repo not found", context do
+    test "return not found message when repo does not exist", context do
       query = """
         repo (slug: "Idont/exist") {
           name
         }
       """
 
       {:ok, resp} = context.conn
-        |> post("/api", AbsintheHelpers.query_skeleton(query))
+        |> execute_query(query)
+        |> decode_response_body
+
+      error_message = Enum.at(resp["errors"], 0)
+
+      assert %{"message" => "not_found"} = error_message
+    end
+
+    test "return required fields message when missing fields", context do
+      required_fields = ["slug"]
+
+      query = """
+        repo {
+          name
+        }
+      """
+
+      resp_body = context.conn
+        |> execute_query(query)
         |> Map.get(:resp_body)
-        |> Antidote.decode
+
+      Enum.each(required_fields, fn field ->
+        assert(
+          Regex.match?(~r/errors.*#{field}.*found null/, resp_body),
+          "Field '#{field}' not found in #{resp_body}"
```

Comment:
this seems a tad to generic, as we are only requiring one field :)

Reply:
I wrote a more specific regex for it.